src/test/resources/features/account/import_account.feature

```diff
@@ -0,0 +1,21 @@
+@Account
+Feature: Import Accounts
+
+  @deleteAccountImports
```

Comment:
Maybe the hook only should be named as "deleteAccount" hook doesn't need to know if the accounts were imported.

Reply:
ohh that is correct sorry